tests/Mediator/constants.xml

```diff
@@ -0,0 +1,164 @@
+        <TERMINATION_COUNTDOWN_IN_SECONDS>0</TERMINATION_COUNTDOWN_IN_SECONDS>
+        <DS_DELAY_WAKEUP_IN_SECONDS>5</DS_DELAY_WAKEUP_IN_SECONDS>
+        <SHARD_DELAY_WAKEUP_IN_SECONDS>10</SHARD_DELAY_WAKEUP_IN_SECONDS>
+        <NUM_FORWARDED_BLOCK_RECEIVERS_PER_SHARD>10</NUM_FORWARDED_BLOCK_RECEIVERS_PER_SHARD>
+        <NUM_OF_TREEBASED_CHILD_CLUSTERS>5</NUM_OF_TREEBASED_CHILD_CLUSTERS>
+        <FETCH_LOOKUP_MSG_MAX_RETRY>3</FETCH_LOOKUP_MSG_MAX_RETRY>
+        <MAX_CONTRACT_DEPTH>5</MAX_CONTRACT_DEPTH>
+        <COMMIT_WINDOW_IN_SECONDS>5</COMMIT_WINDOW_IN_SECONDS>
+        <NUM_CONSENSUS_SUBSETS>5</NUM_CONSENSUS_SUBSETS>
+        <MISORDER_TOLERANCE_IN_PERCENT>5</MISORDER_TOLERANCE_IN_PERCENT>
+        <MAX_CODE_SIZE_IN_BYTES>20480</MAX_CODE_SIZE_IN_BYTES>
+    </constants>
+    <options>
+        <TEST_NET_MODE>false</TEST_NET_MODE>
+        <EXCLUDE_PRIV_IP>false</EXCLUDE_PRIV_IP>
+        <ENABLE_DO_REJOIN>true</ENABLE_DO_REJOIN>
+        <FULL_DATASET_MINE>false</FULL_DATASET_MINE>
+        <OPENCL_GPU_MINE>false</OPENCL_GPU_MINE>
+        <CUDA_GPU_MINE>false</CUDA_GPU_MINE>
+        <LOOKUP_NODE_MODE>false</LOOKUP_NODE_MODE>
+        <ARCHIVAL_NODE>false</ARCHIVAL_NODE>
+        <BROADCAST_GOSSIP_MODE>true</BROADCAST_GOSSIP_MODE>
+        <GOSSIP_CUSTOM_ROUNDS_SETTINGS>true</GOSSIP_CUSTOM_ROUNDS_SETTINGS>
+        <BROADCAST_TREEBASED_CLUSTER_MODE>true</BROADCAST_TREEBASED_CLUSTER_MODE>
+        <GET_INITIAL_DS_FROM_REPO>false</GET_INITIAL_DS_FROM_REPO>
+        <UPGRADE_HOST_ACCOUNT>Zilliqa</UPGRADE_HOST_ACCOUNT>
+        <UPGRADE_HOST_REPO>Zilliqa</UPGRADE_HOST_REPO>
+    </options>
+    <gas>
+        <MICROBLOCK_GAS_LIMIT>500000</MICROBLOCK_GAS_LIMIT>
+        <CONTRACT_CREATE_GAS>50</CONTRACT_CREATE_GAS>
+        <CONTRACT_INVOKE_GAS>10</CONTRACT_INVOKE_GAS>
+        <NORMAL_TRAN_GAS>1</NORMAL_TRAN_GAS>
+        <DEFAULT_MIN_GAS_PRICE>1</DEFAULT_MIN_GAS_PRICE>
+    </gas>
+    <dispatcher>
+        <USE_REMOTE_TXN_CREATOR>false</USE_REMOTE_TXN_CREATOR>
+        <TXN_PATH/>
+    </dispatcher>
+    <archival>
+        <DB_HOST/>
+    </archival>
+    <smart_contract>
+        <SCILLA_ROOT/>
+        <SCILLA_BINARY>bin/scilla-runner</SCILLA_BINARY>
+        <SCILLA_FILES>scilla_files</SCILLA_FILES>
+        <SCILLA_LOG>_build</SCILLA_LOG>
+        <SCILLA_LIB>src/stdlib</SCILLA_LIB>
+        <INIT_JSON>init.json</INIT_JSON>
+        <INPUT_STATE_JSON>input_state.json</INPUT_STATE_JSON>
+        <INPUT_BLOCKCHAIN_JSON>input_blockchain.json</INPUT_BLOCKCHAIN_JSON>
+        <INPUT_MESSAGE_JSON>input_message.json</INPUT_MESSAGE_JSON>
+        <OUTPUT_JSON>output.json</OUTPUT_JSON>
+        <INPUT_CODE>input.scilla</INPUT_CODE>
+    </smart_contract>
+    <accounts>
+        <account>
+            <wallet_address>cc02a3c906612cc5bdb087a30e6093c9f0aa04fc</wallet_address>
+        </account>
+    </accounts>
+    <lookups>
+        <!--IP to be provided after public testnet launch.
```

Comment:
I would like to discuss this part about providing IP. The process has to be tested too.